### PR TITLE
Add home and quotes page

### PR DESCRIPTION
**Pull request by kimothokamau**
This PR adds a home page and a quotes page using the react-router-dom library

### CODE REVIEW

File: src/components/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <div className="navbar-component">
+      <h1>Math Magicians</h1>
+      <nav className="navbar">
+        <ul className="navbar-list">
+          {links.map((link) => (
+            <li key={link.id} className="navbar-list-items">
+              <NavLink to={link.path} activeClassName="active-link" exact>
+                {link.text}
+              </NavLink>
+              <span className="pipe-divisor"> | </span>
```
Comment (Davidosky007) [inline src/components/NavBar.js:33]: * Kindly make your webpage match the given wireframe in the project requirement the links should be aligned to the right 
* YOURS 
image
* ORIGINAL 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @kimothokamau 👋, 

Good job so far! 💪 
* The repo looks professional.✔️
 * Git/GitHub flow best practices have been followed✔️
* There are no linter errors✔️
* You have descriptive PR✔️
## There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/NavBar.js (hunk 1 above)
Comment (kimothokamau) [reply src/components/NavBar.js:33]: Thanks for the review, I have worked on the changes and requested another review

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kimothokamau** [comment]
Thank you @mikenath223 

Happy coding too :wave:

**Pull request merged**